README.md

```diff
@@ -120,6 +121,108 @@ engine.listen(process.env.port || 8000);
 
 Note that this feature is currently in beta which means that it is close to production-ready but has not been run in production yet. We appreciate all efforts to try this out and provide feedback.
 
+
+## Live Mixing (BETA)
+
+This feature gives the possibility to mix in a true live stream in a Channel Engine powered linear channel (VOD2Live).
+
+![RFC Drawing of Live-Mixing Functionality](rfc/channel_engine_vod_with_live.png)
+
+A beta-version of live-mixing with high availability support is available in the Channel Engine. This allows you to use a new component which can let you break in to a scheduled VOD event or Live stream event at any speficied time on top of the usual vod-to-live content. 
+To enable live-mixing, create a class which implements the following interface.
+
+```
+class MyStreamSwitchManager {
+  getSchedule() -> [ { eventId, assetId, title, type, start_time, end_time, uri, duration } ]
+}
+```
+
+The class's `getSchedule()` function should return a list of event objects in the following format:
+
+```
+{
+  "eventId": {
+      "type": "string",
+      "description": "Generated ID of the event. If not specified a uuid-based eventId will be generated"
+  },
+  "assetId": {
+    "type": "string",
+    "description": "The ID of the asset in the schedule event"
+  },
+  "title": {
+    "type": "string",
+    "description": "Title of the asset"
+  },
+  "type": {
+    "type": "number",
+    "description": "Type of event (1=LIVE and 2=VOD)"
+  },
+  "start_time": {
+    "type": "number",
+    "description": "UTC Start time of the event as a Unix Timestamp (in milliseconds)"
+  },
+  "end_time": {
+    "type": "number",
+    "description": "UTC End time of the event as a Unix Timestamp (in milliseconds)"
+  },
+  "uri": {
+    "type": "string",
+    "description": "The URI to the VOD asset or Live Stream"
+  },
+  "duration": {
+    "type": "number",
+    "description": "The duration of the asset (in milliseconds) NOTE: Not required for Live Stream events"
+  }
+}
+```
+
+Below are examples of a Live stream event and a VOD event respectively:
+```
+{
+  eventId: "eeecd5ce-d2d2-48db-b1b3-233957f7d69e",
+  assetId: "live-asset-4",
+  title: "My scheduled Live stream event",
+  type: 1,
+  start_time: 1631003900000,
+  end_time: 1631003921000,
+  uri: "https://cph-p2p-msl.akamaized.net/hls/live/2000341/test/master.m3u8",
+}
+```
+```
+{
+  eventId: "26453eea-0ac2-4b89-a87a-73d369920874",
+  assetId: "vod-asset-13",
+  title: "My scheduled VOD event",
+  type: 2,
+  start_time: 1631004100000,
+  end_time: 1631004121000,
+  uri: "https://maitv-vod.lab.eyevinn.technology/VINN.mp4/master.m3u8",
+  duration: 2 * 60 * 1000,
+}
```

Comment:
@Nfrederiksen Should we mention that the `end-time` will be ignored for VOD events? Because we are using the `duration` to set that.

Reply:
it not that we ignore endtime, its more that the duration dictates when v2l content will start again. our switcher still uses endtime as a determine whether to be on Session or SessionLive but VOD events are hosted on Session anyway.

i see your point though

Reply:
adding more text

Reply:
Yes exactly, poorly explained by me. I just thought that it might be confusing to both have `end_time` and `duration` for the user without an explanation that the `end_time` is there purely for the stream-switchers sake.